platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/linking/InOrderLinker.java

```diff
@@ -51,6 +62,19 @@ public class InOrderLinker {
      */
     private static final int INITIAL_CAPACITY = 1024;
 
+    /**
+     * The minimum period between log messages for a specific mode of failure.
+     */
+    private static final Duration MINIMUM_LOG_PERIOD = Duration.ofMinutes(1);
```

Comment:
I'm seeing this in a lot of places.  Does this need to be configurable, a default value for rate limited logs?

Reply:
I'm asking the question for discussion purposes, not making a definite change request.

Reply:
@edward-swirldslabs I created an issue for this https://github.com/hashgraph/hedera-services/issues/10299